### ISSUE TITLE
Link naar AP generiek#Identificator resolved niet meer

### ISSUE THREAD

**Issue by madnificent** [Informatievlaanderen/Data.Vlaanderen.be#329]
**Link waar het probleem zich voor doet:** https://data.vlaanderen.be/doc/applicatieprofiel/mandatendatabank/#Persoon

**Omschrijving van het probleem:** De URL https://data.vlaanderen.be/doc/applicatieprofiel/generiek#Identificator waar in de commentaar van de eigenschap Identificator naar verwezen wordt resolved niet meer.

**Omschrijving van een mogelijke oplossing:** Gebruik van cool URIs zodat verwijzingen intern en extern blijven werken.

**Comment by GeertThijs** [comment]
De link is fout, moet zijn: https://data.vlaanderen.be/doc/applicatieprofiel/generiek-basis#Identificator. Momenteel kom je dan wel in de contextfile terecht, op test is dit wel ok, zie https://test.data.vlaanderen.be/doc/applicatieprofiel/generiek-basis#Identificator.

**Comment by madnificent** [comment]
Inderdaad, het applicatieprofiel wordt nu gehost onder generiek-basis.  Heeft de url https://data.vlaanderen.be/doc/applicatieprofiel/generiek#Identificator nooit bestaan of is het applicatieprofiel generiek hernoemd naar generiek-basis?  Ik weet niet goed hoe ik dergelijke informatie in deze repository moet zoeken.

**Comment by GeertThijs** [comment]
Is volgens mij niet veranderd geweest...

**Comment by madnificent** [comment]
Super.  Vreemd dat niemand het opgemerkt heeft.  Herstellen jullie deze in de toekomst of moet ik hier actie voor ondernemen?
